staging/src/k8s.io/cloud-provider/controllers/route/route_controller.go

```diff
@@ -94,7 +94,7 @@ func New(routes cloudprovider.Routes, kubeClient clientset.Interface, nodeInform
 	return rc
 }
 
-func (rc *RouteController) Run(stopCh <-chan struct{}, syncPeriod time.Duration) {
+func (rc *RouteController) Run(ctx context.Context, stopCh <-chan struct{}, syncPeriod time.Duration) {
```

Comment:
I believe this is going to break https://github.com/kubernetes/kubernetes/blob/5772839933748ca7bff3bdf08360d5d56d7b1216/staging/src/k8s.io/cloud-provider/app/core.go#L142.

Reply:
Yeah these have to be updated accordingly.